hi-ml-cpath/testhisto/testhisto/utils/test_viz_utils.py

```diff
@@ -150,11 +149,11 @@ def assert_plot_tiles_figure(tiles_fig: plt.Figure, fig_name: str, test_output_d
 @pytest.mark.skipif(is_windows(), reason="Rendering is different on Windows")
 def test_plot_top_bottom_tiles(slide_node: SlideNode, test_output_dirs: OutputFolderForTests) -> None:
     top_tiles_fig = plot_attention_tiles(
-        case="TP", slide_node=slide_node, top=True, num_columns=4, figsize=(10, 10)
+        case="FN", slide_node=slide_node, top=True, num_columns=4, figsize=(10, 10)
     )
```

Comment:
Because in the case of TP: pred_prob_score is equal to gt_prob_score, I changed it to make sure we catch the bug that we had before. All cases TP/FN... are tested in the tiles selector tests. Here we just want to make sure that the plots are correct. If we add TP as well. we will need to add extra plot files in the repo not sure of it's worth it. what do you think?

Reply:
it would seem we have 4 plots that are testing the same case (FN with P=0.96) - maybe we can diversify? I might miss something here

Reply:
The idea is to check for a given slide_node that we are plotting the expected 4 plots (top tiles, bottom tiles, heatmaps, and slide thumbnail)

Reply:
that's why we have slide_node as fixture to be used in all tests.
I limited the tests to FN scenario only because here we just making sure plots are rendered correctly. All the logic of tiles selection ... is tested elsewhere